src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/Html/HtmlTagCompletionProvider.ts

```diff
@@ -102,10 +102,13 @@ export class HtmlTagCompletionProvider {
             return;
         }
 
-        const languageResponse = await this.serviceClient.languageQuery(lastChange.range.start, document.uri);
-        if (languageResponse.kind !== LanguageKind.Html) {
-            // This prevents auto-completion of things like C# generics.
-            return;
+        if (documentContent.substring(changeOffset - 'text'.length, changeOffset) !== 'text') {
```

Comment:
Move this logic into a method, `if (!AtMarkupTransition()) { ... }`

Reply:
I can update it to `<text` if preferred. I was aiming to support `< text` assuming it would be valid syntax?

Reply:
Nope, not valid Syntax 😄

Reply:
Updated to only support `<text` and not `< text`